src/scss/index.scss

```diff
@@ -2,6 +2,11 @@
   display: none;
 }
 
+body {
+  background-color: #0C090A;
+  color: #ffffff;
+}
```

Comment:
If you use Bootstrap and Sass you can also use variables (like `$primary-color`).